src/pages/ShoppingCart/utils/calculateTotalPrice.ts

```diff
@@ -0,0 +1,13 @@
+import { ShoppingCartProduct } from './concatProductsAndQuantity';
+
+export const calculateTotalPrice = (
+  cartProducts?: ShoppingCartProduct[]
+): string => {
+  const prices = cartProducts?.map(
+    (cartProduct) => cartProduct.product.base_price_sol * cartProduct.quantity
```

Comment:
maybe try to cast to number before using math operators, for example 
```suggestion
    (cartProduct) => Number(cartProduct.product.base_price_sol) * Number(cartProduct.quantity)
```

But I would prefer to make it in a separate line to cover possible errors correctly

Reply:
Of course, thanks for the suggestion